src/GHC/All.hs

```diff
@@ -156,6 +156,10 @@ createModuleEx :: ApiAnns -> Located HsModule -> ModuleEx
 createModuleEx anns ast =
   ModuleEx (applyFixities (fixitiesFromModule ast ++ map toFixity defaultFixities) ast) anns
```

Comment:
Could you redefine `createModuleEx = createModuleExWithFixities defaultFixities`